### PR TITLE
add devcontainer setup and update readme

### PR DESCRIPTION
**Pull request by olevett**
I hit a couple of issues running through the set up so I thought I'd document them and try and make it simpler for someone in future

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/anaconda
```
Comment (galund) [inline .devcontainer/devcontainer.json]: I feel like this isn't the correct readme - that's about some specific anaconda dev container (which I'm quite glad to not have to think about as anaconda is a bit weird)

Comment (olevett) [reply .devcontainer/devcontainer.json]: Interesting, I think I rebuilt this once or twice, I'll just get rid of the readme reference

File: README.md [hunk 2]
```diff
@@ -29,6 +36,10 @@ npm install -g sass  # or your local equivalent per https://sass-lang.com/instal
```
Comment (galund) [inline README.md:36]:
```suggestion

If the above dependencies/versions need to change, please update the [devcontainer configuration](.devcontainer/devcontainer.json) also.

```

**Review by galund — APPROVED** [review]
(Works for me, just made a suggestion to try and keep the readme and the devcontainer in sync.)

**Pull request merged**